### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by alexkeating**
## GitHub Issue

#251 #245 

## Changes

Add some hub navigation mobile friendly
Screen Shot 2022-06-20 at 12 41 37 PM
Screen Shot 2022-06-20 at 12 41 29 PM
Screen Shot 2022-06-20 at 12 41 21 PM

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/hub-app/src/components/Profile.tsx [hunk 1]
```diff
@@ -45,55 +38,20 @@ const AvatarLg = styled(Avatar)`
...
-        size="lg"
-        src={profile.image}
-        alt="profile image"
-        fallback={<StyledGhost />}
-      />
+      <AvatarLg src={profile?.image || ''} alt="profile image" />
       <div>
         <NameContainer>
-          <H5>{profile.name}</H5>
+          <H5>{profile?.name || ''}</H5>
```
Comment (0xRowdy) [inline apps/hub-app/src/components/Profile.tsx:52]: Do you need to render an empty H5 if name doesn't exist?
Something like
```suggestion
         {profile?.name && <H5>{profile?.name}</H5>}
```

File: apps/hub-app/src/components/Profile.tsx [hunk 2]
```diff
@@ -45,55 +38,20 @@ const AvatarLg = styled(Avatar)`
...
           <ParLg as="span" role="img" aria-label="profile emoji">
-            {profile.emoji}
+            {profile?.emoji || ''}
           </ParLg>
```
Comment (0xRowdy) [inline apps/hub-app/src/components/Profile.tsx:55]: Conditionaly render ParLg based on profile?.emoji

File: apps/hub-app/src/components/ConnectCard.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+const ConnectCard = () => {
+  return (
+    <Container>
+      <ParMd>
+        <Bold>Connect Wallet to view Dashboard</Bold>
+      </ParMd>
+      <ParSm>
+        View your DAOs,active proposals, and navigate to vote and interact
```
Comment (0xRowdy) [inline apps/hub-app/src/components/ConnectCard.tsx:63]:
```suggestion
        View your DAOs, active proposals, and navigate to vote and interact
```

File: apps/hub-app/src/components/ConnectCard.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useHausConnect } from '@daohaus/daohaus-connect-feature';
+import { FiMoreHorizontal } from 'react-icons/fi';
+import {
+  Button,
+  H5,
```
Comment (0xRowdy) [inline apps/hub-app/src/components/ConnectCard.tsx:7]: Remove unused

File: apps/hub-app/src/components/BodyNav.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+const BodyNavContainer = styled.div`
+  display: flex;
+  gap: 1rem;
+
+  @media (min-width: ${widthBreakpoint.mobileLg}px) {
+    align-items: center;
+    justify-content: center;
+  }
+`;
```
Comment (0xRowdy) [inline apps/hub-app/src/components/BodyNav.tsx:29]: Consistant spacing after
```suggestion
`;

```

File: apps/hub-app/src/components/BodyNav.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+const StyledLink = styled(Link)`
+  text-decoration-color: ${crimsonDark.crimson11};
+  color: ${crimsonDark.crimson12};
+  text-underline-offset: 0.7rem;
+`;
```
Comment (0xRowdy) [inline apps/hub-app/src/components/BodyNav.tsx:34]:
```suggestion
`;

```

**Review by 0xRowdy — APPROVED** [review]

**Review by skuhlmann — APPROVED** [review]

File: apps/hub-app/src/components/Profile.tsx (hunk 1 above)
Comment (alexkeating) [reply apps/hub-app/src/components/Profile.tsx:52]: Happy to change it is there any visual difference? Is it mainly to remove markup clutter

Comment (0xRowdy) [reply apps/hub-app/src/components/Profile.tsx]: It's  two fold.
You'll get an "Empty Heading Tag" accessibility checkers. A screen reader will alert a user that there is an empty tag, h5 in this case and stop. The user has to progress to the next element.
Second is to keep the markup cleaner.

If you get an error you can covert to a boolean
`{!!item.mystring && <H5>...`
Or use a ternary operator and return null.
```
{item.mystring ?
 <H5>...
  :
  null
}
```

**Pull request merged**